queries/agenda.js

```diff
@@ -63,6 +66,7 @@ const fetchFilesFromAgendaByMandatees = async (agendaId, mandateeIds, currentUse
   WHERE {
       ?agendaitem a besluit:Agendapunt ;
           besluitvorming:geagendeerdStuk ?document .
+      FILTER NOT EXISTS { [] pav:previousVersion ?document }
```

Comment:
same here, side effect if a nextVersion exists on a later agenda. same fix of double `besluitvorming:geagendeerdStuk` and changed `filter not exists`